From bb8040802bd91d7847b82179cd66de4f8e92912b Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megous@megous.com>
Date: Fri, 12 Apr 2019 01:52:54 +0200
Subject: [PATCH 03/29] dt-bindings: Add compatible for H6 RTC

RTC on H6 is similar to the one on H5 SoC, but incompatible in small
details. See the driver for description of differences. Add new
compatible for this RTC.

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 Documentation/devicetree/bindings/rtc/sun6i-rtc.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/rtc/sun6i-rtc.txt b/Documentation/devicetree/bindings/rtc/sun6i-rtc.txt
index 6b732c4..291505b 100644
--- a/Documentation/devicetree/bindings/rtc/sun6i-rtc.txt
+++ b/Documentation/devicetree/bindings/rtc/sun6i-rtc.txt
@@ -11,6 +11,7 @@ Required properties:
 		    - "allwinner,sun8i-v3-rtc"
 		    - "allwinner,sun50i-a64-rtc", "allwinner,sun8i-h3-rtc"
 		    - "allwinner,sun50i-h5-rtc"
+		    - "allwinner,sun50i-h6-rtc"
 
 		  Where there are two or more compatible strings, this
 		  denotes the hardware covered by the most specific one
-- 
2.7.1

